metrotranscom/doorway
# feat: creating oakland ami chart

# Pull Request Template

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/3588)


## Description
Creates new oakland ami chart

## How Can This Be Tested/Reviewed?
run the new migration locally, you should be able to use the new oakland ami chart when creating a new unit

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
